tests/unit/Storefront/Controller/CheckoutControllerTest.php

```diff
@@ -186,18 +186,6 @@ public function testGetCartJson(): void
         static::assertEquals($cart, $response->getObject());
     }
 
-    public function testConfirmPageNoCustomer(): void
```

Comment:
can we instead check this with an integration test? :thinking:

Reply:
Yes, we can :-) But I checked some other places, and there were no similar redirect tests for `_loginRequired`, are you aware of a place where this is checked?

Otherwise I can think of one myself, but need to see when I can implement it.

Reply:
I hope, that we test this functionality somewhere :sweat_smile: :see_no_evil: but I haven't had a look yet

Reply:
I have not found any, but I implemented some for the changes :-)

Reply:
I will check why the tests are failing in the CI, they work locally for me.

Reply:
Fixed it :-)